packages/aor-graphql-client/src/index.js

```diff
@@ -58,14 +57,15 @@ export default async options => {
 
         const { parseResponse, ...query } = overridedbuildQuery
             ? {
-                ...buildQuery(aorFetchType, resource, params),
-                ...overridedbuildQuery(params),
-            }
+                  ...buildQuery(aorFetchType, resource, params),
+                  ...overridedbuildQuery(params),
+              }
             : buildQuery(aorFetchType, resource, params);
 
         if (QUERY_TYPES.includes(aorFetchType)) {
             const apolloQuery = {
                 ...query,
+                fetchPolicy: 'network-only',
```

Comment:
To make sure I got this right, with this change you aren't using the Apollo cache for anything?

Reply:
That's right: AOR already has its own cache.

Reply:
Oh yeah, forgot about the AOR cache. Cool deal, yeah this change makes sense.